test/Driver/Dependencies/only-skip-once.swift

```diff
@@ -9,7 +9,7 @@
 // CHECK-INITIAL: {{(Job finished: {compile: main.o <= main.swift}|Finished Compiling main.swift)}}
 // CHECK-INITIAL: {{(Job finished: {compile: file1.o <= file1.swift}|Finished Compiling file1.swift)}}
 // CHECK-INITIAL: {{(Job finished: {compile: file2.o <= file2.swift}|Finished Compiling file2.swift)}}
-// CHECK-INITIAL: {{(Job finished: {link: main <= main.o file1.o file2.o}|Finished Linking main)}}
+// CHECK-INITIAL: {{(Job finished: {link: main <= main.o file1.o file2.o|Finished Linking main)}}
```

Comment:
This test now passes on linux as part of the early swift driver tests, so I enabled it, but it then failed with the old C++ Driver.

The issue is that these two lines don't expect another autolink file to be added on macOS, but that is needed on linux, which outputs this instead:
```
Job finished: {link: main <= main.o file1.o file2.o ... 1 more}
```
Omitting the closing brace here and below should get this test to work on both Mac and linux.